Cargo.lock

```diff
@@ -21,6 +21,7 @@ dependencies = [
  "serde 0.6.15 (registry+https://github.com/rust-lang/crates.io-index)",
  "serde_json 0.6.0 (registry+https://github.com/rust-lang/crates.io-index)",
  "serde_macros 0.6.14 (registry+https://github.com/rust-lang/crates.io-index)",
+ "sha1 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
I just learned that sha1 implementation is not great (it does extra allocations). On the other hand https://github.com/DaGenix/rust-crypto/ looks better.